settings.php

```diff
@@ -165,6 +165,42 @@
             get_string('spmetadatasign_help', 'auth_saml2'),
             0, $yesno));
 
+    $settings->add(new admin_setting_configtext(
+        'auth_saml2/entityid',
+        get_string('entityid', 'auth_saml2'),
+        get_string('entityid_help', 'auth_saml2'),
+        "$CFG->wwwroot/auth/saml2/sp/metadata.php"
```

Comment:
shouldn't the default be blank? - eg what happens when a prod site gets set up and then sync to test... you end up with a testing site pointing at prod site metadata.php accidentally....

Reply:
yep, set to '' now